packages/web-components/src/components/cbp-list/cbp-list.tsx

```diff
@@ -0,0 +1,45 @@
+import { Component, Host, Element, Prop, h } from '@stencil/core';
+import { setCSSProps } from '../../utils/utils';
+
+@Component({
+  tag: 'cbp-list',
+  styleUrl: 'cbp-list.scss',
+})
+export class CbpList {
+  @Element() host: HTMLElement;
+  
+  private renderedTag: HTMLElement;
+
+  /** Specifies the semantic tag to be rendered. */
+  @Prop() tag!: 'ul' | 'ol' ;
+
+  /** Supports adding inline styles as an object */
+  @Prop() sx: any = {};
+
+  /** Specifies the context of the component as it applies to the visual design and whether it inverts when light/dark mode is toggled. Default behavior is "light-inverts" and does not have to be specified. */
+  @Prop({ reflect: true }) context: "light-inverts" | "light-always" | "dark-inverts" | "dark-always";
+  
```

Comment:
This component also needs an `accessibilityText` prop that renders and `aria-label` on the list. See structured list reqs and add it the same guidance to these requirements.